Test/VariantSearchTests.cs

```diff
@@ -213,106 +219,221 @@ public void VariantSpecificOutputFiles()
 
             Assert.AreEqual("Number of potential variant containing peptides identified at 1% FDR: 1", checkResults[4]);
             Assert.AreEqual("Number of unqiuely identified variant peptides at 1% FDR: 1", checkResults[5]);
-            Assert.AreEqual("Number of SAV variant peptides at 1% FDR: 0", checkResults[6]);
-            Assert.AreEqual("Number of frameshift variant peptides at 1% FDR: 1", checkResults[7]);
-            Assert.AreEqual("Number of inframe insertion variant peptides at 1% FDR: 0", checkResults[8]);
-            Assert.AreEqual("Number of inframe deletion variant peptides at 1% FDR: 0", checkResults[9]);
-            Assert.AreEqual("Number of stop gain variant peptides at 1% FDR: 0", checkResults[10]);
-            Assert.AreEqual("Number of stop loss variant peptides at 1% FDR: 0", checkResults[11]);            
-            Assert.AreEqual("Number of modified variant peptides at 1% FDR: 1", checkResults[12]);
-            Assert.AreEqual("Number of modified variant sites at 1% FDR: 0", checkResults[13]);
+            Assert.AreEqual("Number of unique variants: 1", checkResults[6]);
```

Comment:
Nice test. 

Could consider making these a static list of strings in the postprocessing class, so that you don't have to change them more than once. Could loop through that list and a list of expected results here.